src/middleware/validate.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response, NextFunction } from 'express';
+import { validationResult } from 'express-validator';
+
+// Middleware to automatically send 442 response with errors on failed validation
+const validate = (req: Request, res: Response, next: NextFunction) => {
+  const errors = validationResult(req);
+  if (errors.isEmpty()) {
+    return next();
+  }
+
+  return res.status(422).json({ errors: errors.array() });
```

Comment:
We can customize how we want to return validation errors for all routes here

Reply:
Dammmmmmn